components/Filter/styles.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import { FilterHeadingUnused } from '@/styles/texts';
+
+export const FilterBackgroundStyles = styled.div<{ isActive: boolean }>`
+  margin-right: 2%;
```

Comment:
i think you have to make the changes here and below where I commented. The styling still does not look like josh's figma hi-fis